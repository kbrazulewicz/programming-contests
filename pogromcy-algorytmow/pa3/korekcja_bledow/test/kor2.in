16
a 0000000
b 0001111
c 0010110
d 0011001
e 0100101
f 0101010
g 0110011
h 0111100
i 1000011
j 1001100
k 1010101
l 1011010
m 1100110
n 1101001
o 1110000
p 1111111
5
10101011110000001100101111000000000110011011001101000011110100101100110000000
011110001001011001100001000010110101000011001011010011001110000
011110001001011001100000111101100000001111
000111100000000010110000000000011110000000000111100000000011001000000011001100000100
0000000000111100101100011001010010101010100110011011110010000111001100101010110110101100110110100111100001111111

