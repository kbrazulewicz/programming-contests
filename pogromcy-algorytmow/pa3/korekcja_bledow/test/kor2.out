OK
kodhamminga
OK
hejalicjo
OK
hejbob
OK
bacababadama
OK
abcdefghijklmnop
